enhancements/update/clusterversion-osus-signatures.md

```diff
@@ -0,0 +1,138 @@
+---
+title: ClusterVersion-has-option-to-reach-local-OSUS-instance-for-signature-verification
+authors:
+  - "@PratikMahajan"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@wking"
+approvers:
+  - "@sdodson"
+api-approvers:
+  - N/A
+creation-date: 2023-09-27
+last-updated: 2023-09-28
+tracking-link:
+  - https://issues.redhat.com/browse/OTA-916
+---
+
+# ClusterVersion has option to reach local OSUS instance for signature verification
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] Operational readiness criteria is defined
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+CVO currently fetches the upstream urls for signature verification through [cluster-update-keys]. We want to have a way 
+in the disconnected and connected environments to have a custom image verification URI to verify the release images.
+
+
+## Motivation
+
+Today, in the disconnected environments, a user has to clone the signatures config map as well as the release images to update the clusters.
+With [serve image signatures for disconnected environments], OSUS will be able to serve the cloned image signatures locally
+
+There is no existing way in CVO to fetch the signatures through a custom URI. This enhancement defines an API change in the CVO
+to accept a custom update-service URI which can be used to verify release signatures. 
+
+
+### User Stories
+
+* As a cluster administrator, I want the cluster's CVO to add a new signature store URI.
+
+* As a cluster administrator, I want the cluster's CVO to replace the existing signature store URIs with a user provided URI
+
+### Goals
+
+* To verify the release image signature in disconnected environments using OSUS signatures URI.
+* To replace existing signature stores with custom user-provided stores.
+* Implementation will require an API change.
+
+### Non-Goals
+
+* We will not verify the authenticity of the verification signature source.
```

Comment:
We check signature validity, regardless of where we find those signatures.  What would this kind of source/store verification look like?